Ic solutions info theory

patriciomassaro opened the pull request.

patriciomassaro commented:
![](https://github.trello.services/images/mini-trello-icon.png) [IC Solutions SL - Sheet 5](https://trello.com/c/nK3cJ1m3/232-ic-solutions-sl-sheet-5)

ludwigbothmann commented on exercises/information-theory/ex_rnw/ic_sol_entropy_binary_classification.Rnw:
```diff
@@ -0,0 +1,76 @@
+  
+  \begin{itemize}
+    \item
+      As $\epsilon$ and $x$ are have a deterministic relation, the uncertainty introduced by the $\psi$ function is 0. The entropy of $\epsilon$ is given only by the entropy of $x$
```
delete "are"

ludwigbothmann reviewed (changes requested):
Please also add the pdfs in exercise-pdf in the end

patriciomassaro commented on exercises/information-theory/ex_rnw/ic_sol_entropy_binary_classification.Rnw:
```diff
@@ -0,0 +1,76 @@
+    The conditional entropy quantifies the uncertainty of $y$ if the outcome of $x$ is given. It is defined as the expected value of the entropies of the conditional distributions, averaged over the conditioning random variable.
+    
+    \begin{align*}
+      H(y|x) &= \E_x [H(y|x=x)] = \sum_{x \in \Omega_x } H(y|x=x) \\
```
Yes, you are right. But I wrote $P(x=x)$ instead, as theta is already defined as the probability that x = 1

patriciomassaro commented:
- PDFs added
- Solved the typos
- Instead of adding a theta, I added something different after the sum, please take a look at that and let me know your opinion.

patriciomassaro commented:
- I moved the Entropy exercise to the second IC exercise sheet of Info Theory
- I added the KL divergence exercise to the first IC exercise sheet and uploaded a draft for the solution.
